addons/website/static/src/scss/website.scss

```diff
@@ -190,6 +190,16 @@ $-seen-urls: ();
     }
 };
 
+html {
```

Comment:
Instead of creating yet another rule for the browser to search for, you could add this in the `:root` definition just above.